src/web/lib/getCommentContext.ts

```diff
@@ -22,17 +22,77 @@ type CommentContextType = {
     discussionKey: string;
     discussionWebUrl: string;
     discussionApiUrl: string;
-    orderBy: 'oldest' | 'newest' | 'mostrecommended';
-    pageSize: 20 | 25 | 50 | 100; // TODO: Review these https://trello.com/c/7v4VDNY0/1326-review-page-size-values
+    orderBy: OrderByType;
+    pageSize: PageSizeType; // TODO: Review these https://trello.com/c/7v4VDNY0/1326-review-page-size-values
     page: number;
 };
 
+type OrderByType = 'newest' | 'oldest' | 'mostrecommended';
+type ThreadsType = 'collapsed' | 'expanded' | 'unthreaded';
+type PageSizeType = 25 | 50 | 100;
+
+interface FilterOptions {
+    orderBy: OrderByType;
+    pageSize: PageSizeType;
+    threads: ThreadsType;
+}
+
+const objAsParams = (obj: any): string => {
```

Comment:
feel like this could be in lib, it might come in handy elsewhere

Reply:
It might, but for now it's only used in one place and having the code co located is a big bonus to readability. I'm reluctant to abstract away if not needed.